sdcm/nemesis.py

```diff
@@ -749,7 +751,10 @@ def toggle_table_ics(self):
                 new_compaction_strategy_as_dict.update(param)
         cmd = "ALTER TABLE {keyspace_table} WITH compaction = {new_compaction_strategy_as_dict};".format(**locals())
         self.log.debug("Toggle table ICS query to execute: {}".format(cmd))
-        self.target_node.run_cqlsh(cmd)
+        try:
+            self.target_node.run_cqlsh(cmd)
```

Comment:
I would check the stout for "Unable to find compaction strategy class 'IncrementalCompactionStrategy'" since the command can fail because of other reasons...